controller/event/authentication.go

```diff
@@ -0,0 +1,111 @@
+/*
+	Copyright NetFoundry Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package event
+
+import (
+	"fmt"
+	"time"
+)
+
+const (
+	AuthenticationEventNS = "authentication"
+
+	AuthenticationEventTypeFail    = "fail"
+	AuthenticationEventTypeSuccess = "success"
+)
+
+// An AuthenticationEvent is emitted when an authentication attempt is made
+//
+// Types of authentication events
+//   - failed - authentication failed
```

Comment:
failed or fail? is this supposed to be the same as what's on line 27?

Reply:
Ah yea, I chagned the value w/o changing the doc.